### DISCUSSION TITLE
The alternative solution to installation

### DISCUSSION THREAD

**Discussion by Krysme** [rustdesk/rustdesk#4299] [General]
On Windows, the portable version of RustDesk sometimes encounters the "No Display" error on certain machines, as indicated by the software itself.

I'm wondering if there's an alternative solution to make the portable version work, similar to installing it. I'm also curious about the technical cause behind this error.

Any hints or information regarding the source code location would be greatly appreciated. Thank you in advance for your assistance.

**Comment by SergeyMy** [comment]
checked on the nightly assembly?
https://github.com/rustdesk/rustdesk/releases/tag/nightly
PS
release 1.1.9 hopelessly outdated
